src/Social/Messages/Casts/Json.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Social\Messages\Casts;
+
+use Illuminate\Contracts\Database\Eloquent\CastsAttributes;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Str;
+
+class Json implements CastsAttributes
```

Comment:
@FredPeal i think laravel already has the csat json by default https://github.com/bakaphp/kanvas-ecosystem-api/blob/development/src/Kanvas/CustomFields/Models/AppsCustomFields.php#L35

Reply:
Yes, but I created this cast because the message field can be a JSON o plain text and I need check this when the record is fetch